src/index.ts

```diff
@@ -0,0 +1,10 @@
+import * as express from "express";
+import * as path from "path";
+const app = express();
+const port = 3000;
+
+app.use("/static", express.static(path.join(__dirname, "/static")));
+
+app.listen(3000, () => {
+    console.log("ladksjflks");
```

Comment:
Can you use a more-informative message for the server listening? Something like what port is being listened on would be nice!

```suggestion
    console.log(`Listening on port ${port}`);
```